.github/workflows/publish.yml

```diff
@@ -94,16 +91,15 @@ jobs:
                   name: release
                   path: dist/
 
+            - name: Publish distribution on PyPI
+              uses: pypa/gh-action-pypi-publish@release/v1
+              with:
+                  user: __token__
+                  password: ${{ secrets.PYPI_API_TOKEN }}
+
             - uses: softprops/action-gh-release@v0.1.15
               name: Create release
-              if: startsWith(github.ref, 'refs/tags/v')
               with:
                   files: |
                       dist/*
                   generate_release_notes: true
-
-            - name: Publish distribution on PyPI
-              uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
It certainly looks like we would publish to PyPI even if the tag did not start with `v`. I think the new code is correct. Also I've switched the order of PyPI publish and GitHub release. This was, if the PyPI release fails for any reason, we do not create a release on Github.